### PR TITLE
Add a reporter config to image-mirroring-edge-infrastructure

### PR DESCRIPTION
**Pull request by lranjbar**
Adding a reporter config to report to a slack channel giving us a little more visibility on failures of this job.

FYI @osherdp @eliorerz @sagidayan @mkowalski

### CODE REVIEW

**Review by osherdp — APPROVED** [review]

File: ci-operator/jobs/infra-image-mirroring.yaml [hunk 1]
```diff
@@ -1052,6 +1052,15 @@ periodics:
     ci.openshift.io/area: edge-infrastructure
     ci.openshift.io/role: image-mirroring
   name: periodic-image-mirroring-edge-infrastructure
+  reporter_config:
+    slack:
+      channel: '#forum-edge-infra-alerts'
+      job_states_to_report:
+      - failure
+      - error
+      report_template: '{{ if eq .Status.State "success" }} :green_jenkins_circle:
```
Comment (hongkailiu) [inline ci-operator/jobs/infra-image-mirroring.yaml:1061]: The if-part seem never true if `success` is not included in `job_states_to_report`.

Comment (lranjbar) [reply ci-operator/jobs/infra-image-mirroring.yaml]: Ah yeah this was adapted from a config we used to use that was more noisy. I updated it :)

**Pull request merged**